cmd/build/helmify/replacements.go

```diff
@@ -119,6 +119,24 @@ var replacements = map[string]string{
     {{- end }}
     resources:
     - '*'
```

Comment:
does this "*" need to come out of here?

Reply:
No, the star refers to all non-sub-resources. For example "configmaps" or "secrets" or "pods". We still definitely want to trigger validation on those like we have done historically.

This PR does not reduce the amount of resources that gatekeeper validates; it expands that list to cover subresources.